tests/appsec/waf/test_addresses.py

```diff
@@ -514,3 +515,42 @@ class Test_Lambda:
 
     def test_main(self):
         assert False, "Need to write a test"
+
+
+@features.appsec_request_blocking
+@features.grpc_threats_management
+@scenarios.appsec_custom_rules
+class Test_GrpcServerMethod:
+    """Test as a custom rule until we have official rules for the address"""
+
+    def validate_span(self, span, appsec_data):
+        tag = "rpc.grpc.full_method"
+        if not tag in span["meta"]:
+            logger.info(f"Can't find '{tag}' in span's meta")
+            return False
+
+        expected = span["meta"][tag]
+        value = appsec_data["triggers"][0]["rule_matches"][0]["parameters"][0]["value"]
+        if value != expected:
```

Comment:
wow - you like taking risks parsing the payload 😆
really really great assertion thanks a lot